functions/consumer/tcp-consumer/src/test/java/org/springframework/cloud/fn/consumer/tcp/L4Tests.java

```diff
@@ -25,8 +24,7 @@
 /**
  * @author Gary Russell
  */
-@TestPropertySource(properties = { "tcp.consumer.encoder = L4" })
-@Disabled
+@TestPropertySource(properties = {"tcp.consumer.encoder = L4"})
```

Comment:
I did not intentionally reformat the brackets, the spring-cloud-build formatter did this.